.github/workflows/fuzz-nightly.yml

```diff
@@ -0,0 +1,67 @@
+# Runs fuzzing nightly.
+name: fuzz-nightly
```

Comment:
Should we have a separate job that tests based on what was touched in a PR?

Reply:
Not sure. My personal take is nightly task is enough.